editor/code_editor.cpp

```diff
@@ -1216,6 +1216,11 @@ void CodeTextEditor::move_lines_up() {
 				return;
 			}
 
+			// Prevent moving last line with no character selected
+			if (to_column == 0 && i == to_line) {
```

Comment:
Should we be taking one from `to_line` rather then skipping, otherwise below this loop, it looks like `to_line_up` will be off by one?